src/server/scripts/Northrend/Gundrak/boss_moorabi.cpp

```diff
@@ -125,41 +138,49 @@ class boss_moorabi : public CreatureScript
                 }
             }
 
-            void ExecuteEvent(uint32 eventId) override
+            void UpdateAI(uint32 diff) override
             {
-                switch (eventId)
+                if (!me->IsInCombat())
+                    scheduler.Update(diff);
```

Comment:
You can left if is not linked in yourself
linked_respawn will prevent these creature respawn if Moorabi is dead. People use it to avoid some bosses respawn (linking a creature to youself).
If not this case, dont need remove